app/javascript/utils/Export.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Called when Exporting to a CSV for a notes field
+ * Removes all the values that would cause an issue when opening in excel
+ */
+function removeFormulaStart(value) {
```

Comment:
are all of these steps necessary for preventing the csv file from opening properly in excel? it seems like the sanitation done in the back end for this only removes '=' and ' =='

https://github.com/SaraAlert/SaraAlert/blob/master/app/lib/excel_sanitizer.rb#L9

Reply:
When opening in excel it seems like those characters could cause issues sometimes.
![image](https://user-images.githubusercontent.com/7842704/114881692-3c47db80-9dd1-11eb-91b3-09d4aa89b0cf.png)

Reply:
thanks for the explanation!